src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -91,6 +91,12 @@ public void DisabledProtocols_SetSslProtocols_ThrowsException(SslProtocols disab
         [InlineData(SslProtocols.Tls12, true)]
         public async Task GetAsync_AllowedSSLVersion_Succeeds(SslProtocols acceptedProtocol, bool requestOnlyThisProtocol)
         {
+            if (ManagedHandlerTestHelpers.IsEnabled)
+            {
+                // TODO #21452: The managed handler is failing.
+                return;
+            }
```

Comment:
From Diego PR (failed in CI):
https://mc.dot.net/#/user/Diego-Perez-Botero/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/9d042fa1e1c3a3c6554600f1770d1bac8ff813b5/workItem/System.Net.Http.Functional.Tests

Looks like Win7 platform.  It is possible it is an SslStream bug or a complex SCHANNEL Windows OS bug